### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by abhijeetd700**
@anilrao316

Screenshots for assignments 1 & 4 output.....

1) Assignment-1
Screenshot (165)

2) Assignment-4
Screenshot (166)

Assignments 2 & 3 are explained via comments in the file.

### CODE REVIEW

File: Week-3/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "week-3",
```
Comment (anilrao316) [inline Week-3/package.json]: remove this file.. i guess not required

**Comment by anilrao316** [comment]
rest looks good u can merge!

**Comment by anilrao316** [comment]
And going forward let maintain seperate commits for each assignments.. this time its fine

**Comment by abhijeetd700** [comment]
> And going forward let maintain seperate commits for each assignments.. this time its fine

Sure.....

**Pull request merged**